docs/install/authentication-mechanisms-gcp.md

```diff
@@ -131,3 +131,121 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
+     Google Cloud Service Account `cloud-run-events`, 
+     and the Kubernetes Service Account `controller` in namespace `cloud-run-events`.
+     - If you are using Kubernetes Secret for the Control Plane, 
+     refer [here](./authentication-mechanisms-gcp.md/#kubernetes-secrets) to check the 
+     Kubernetes Secret `google-cloud-key` in namespace `cloud-run-events`.
+     
+   ***Note:*** For Kubernetes Secret, it is also possible that the JSON private key is no longer existing under your Google Cloud Service Account 
+   `cloud-run-events`. Then, even the Google Cloud Service Account `cloud-run-events` has all required permissions, and 
+   the corresponding Kubernetes Secret `google-cloud-key ` is in namespace `cloud-run-events`, you still get `PermissionDenied` error. 
+   To such case, you have to re-download the JSON private key and re-create the Kubernetes Secret.
+     
+* ***Resources are READY, but can't receive Event***
+
+   Sometimes, a resource instance is READY, but it can't receive Event. It might be authentication configuration problem, 
+   if the underlying `deployment` doesn't have minimum availability.
+   
+   You can use the following command to check a `deployment`'s available pod is zero.
+   
+   ```shell
+   kubectl get deployment -n resource-instance-namespace
+   ```
+   
+   To solve this issue, you can:
+   - Check the Google Cloud Service Account `cre-pubsub` for the Data Plane has all required permissions.
+   - Check authentication configuration is correct for this resource instance. 
+     - If you are using Workload Identity for this resource instance, 
+     refer [here](./authentication-mechanisms-gcp.md/#workload-identity) to check the
+     Google Cloud Service Account `cre-pubsub`, 
+     and the Kubernetes Service Account in namespace where this resource instance resides.
+     - If you are using Kubernetes Secret for this resource instance, 
+     refer [here](./authentication-mechanisms-gcp.md/#kubernetes-secrets) to check the 
+     Kubernetes Secret `google-cloud-key` in namespace where this resource instance resides.
+     
+   ***Note:*** For Workload Identity, there is a known issue [#759](https://github.com/google/knative-gcp/issues/759) 
+   for credential sync delay (~1min) in resources' underlying components. You'll probably encounter this issue, 
+   if you immediately send Events after you finish Workload Identity setup for a resource instance.
+   
+* ***Resources are not READY, due to WorkloadIdentityReconcileFailed***
+   
+   This error only exists when you use default scenario for Workload Identity. 
+   You can find detail failure reason by:
+   ```shell
+   kubectl describe resource-name resource-instance-name -n resource-instance-namespace
+   ```
+   If the reason related on permission denied, refer to common issue ***Resources are not READY*** for solution.
+   
+   If the reason related on concurrency issue, the controller will retry it in the next reconciliation loop 
```

Comment:
Include an example of what this looks like in the output of the command you had them run.

Reply:
added